### DISCUSSION TITLE
SFTPGO behind a AWS ALB and WAF

### DISCUSSION THREAD

**Discussion by itguy2023** [drakkan/sftpgo#1350] [General]
I started looking at SftpGo a few weeks back and thinking about using it for the company I work for.
I'm using the SFTPGo professional AMI from AWS Market Place.
Its behind a AWS ALB and WAF. 
I can navigate to the "/web/admin/login" page and login in works.
If I try an add a use I get a "Forbidden Error" 503 ?
Is there any documentation on how to configure SFTPGO to work behind AWS ALB and WAF ?

**Comment by drakkan** [comment]
Hello,

If you use SFTPGo from the AWS Marketplace, you are eligible for basic email support. Please write your question to the dedicated support address that you can see on the Marketplace page including your AWS account ID so we can verify your subscription. Thank you

**Comment by itguy2023** [comment]


**Reply by drakkan** [reply]
It is in the support section of the Marketplace page

![Schermata del 2023-07-07 12-12-56](https://github.com/drakkan/sftpgo/assets/553263/908915d7-6ace-47cc-a3fd-93627c58e3e5)

Please write to the address above and also attach/check the sftpgo logs to understand if the request reaches SFTPGo or if the 503 error is generated elsewhere. Logs are written to the directory "/srv/sftpgo/logs". Thank you for using SFTPGo from the Marketplace

**Comment by itguy2023** [comment]
Resolved
